cleanslate/cleanslate/urls.py

```diff
@@ -20,6 +20,7 @@
 from django.conf import settings
 from django.conf.urls.static import static
 from doma.views import home, profile, calendar, reminders, finance, edit_chore_deadline, create_chore, delete_chore, EditUserProfileView
+from django.contrib.auth import views as auth_views
```

Comment:
Say this in the other PRs, why do we need this?

Reply:
This is for the code that is currently being worked on. 
https://docs.djangoproject.com/en/1.11/topics/auth/default/

Reply:
`urlpatterns += [
    url(r'^doma/', include('django.contrib.auth.urls'))
]`
on line 45 should have this covered?

Reply:
I have removed this line in my PR as I don't see that it is necessary